exercises/02-publish-subscribe/instructions.md

```diff
@@ -141,7 +141,7 @@ routing.RegisterPublisher(typeof(OrderSubmittedEvent), "Divergent.Sales");
 
 ### Step 8
 
-When Finance receives the `OrderSubmittedEvent` message it needs to keep track of item prices that belong to the submitted order. And finally initiate the payment process by sending the `InitiatePaymentProcessCommand` message.
+When Finance receives the `OrderSubmittedEvent` message it records prices for the items that belong to the submitted order. And finally initiate the payment process by sending the `InitiatePaymentProcessCommand` message.
```

Comment:
Should we back-up the claim that we `need to keep track of item prices`?

Reply:
Good point. The idea is that items put in the shoppingcart for a certain price, remain in the cart for that price, even though the price might change during shopping.
This is a nice way to display how Sagas simplify this. Backing up the claim, for example/among others in the section `business requirements` will enhance this.